petastorm/pyarrow_helpers/tests/test_batch_buffer.py

```diff
@@ -145,7 +145,7 @@ def test_random_table_size_and_random_batch_sizes():
             if not batcher.empty():
                 read_batch = batcher.get()
                 for value in compat_column_data(read_batch.columns[0]):
-                    assert value == read_seq
+                    assert value.as_py() == read_seq
```

Comment:
value is a pyarrow object now and an explicit conversion to a python type is now needed.